sensor_msgs/LaserScan scan
geometry_msgs/Pose[] agent_poses
std_msgs/Float32MultiArray[] distance_matrices
float32 reward
bool done
---
geometry_msgs/Twist action
bool success
